app/jobs/sync_mailing_lists_job.rb

```diff
@@ -31,6 +30,9 @@ def perform
     GsuiteMailingLists.sync_group("sports@worldcubeassociation.org", Team.wsot.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("archive@worldcubeassociation.org", Team.wat.current_members.includes(:user).map(&:user).map(&:email))
 
+    treasurers = UserGroup.officers[0].active_roles.filter { |role| role.metadata.status == 'treasurer' }
```

Comment:
Do we have a constant somewhere that we can reference, instead of writing the raw string `'treasurer'` here?

Reply:
Also, why `officers[0]`?

Reply:
Also, after `active_roles` you can include `.includes(:user)` for more efficient loading ;)

Reply:
Yes there was a constant, I've added that. I added officers[0] because `UserGroup.officers` was returning list of groups. I've now changed it to consider every officer groups even though there is only one. I hope that is fine.

`.includes(:user)` won't work I guess, because `active_roles` is not yet `ActiveRecord`, it's still an array. Maybe after the migration is completed?

Reply:
Ah, yes. In that case my brain was already thinking too far ahead. Sorry! :innocent: